src/js/routes/nodes.js

```diff
@@ -45,12 +46,8 @@ let nodesRoutes = {
         {
           type: Route,
           name: 'nodes-task-panel',
-          path: 'task-detail/:taskID/?'
-        },
-        {
-          type: Route,
-          name: 'node-detail-tasks',
-          path: 'task-detail/:taskID/?'
+          path: 'task-detail/:taskID/?',
+          handler: TaskDetail
```

Comment:
It is wrapped in something like that.
But I've disabled the breadcrumbs for the node pages, because the need to be generalized first.